dolfyn/adv/tools.py

```diff
@@ -0,0 +1,34 @@
+import dolfyn.adv.turbulence as turb
+import numpy as np
+
+
+def t_set(advr, n_bin, n_fft=None):
+    # note, can get more accurate t_range by reducing the bin size
```

Comment:
Also, should I create a json_reader branch to work on those changes or just add them to the jupyter_examples branch?